src/main/java/com/wolApi/wolApi/BibleVerseController.java

```diff
@@ -1,26 +1,35 @@
 package com.wolApi.wolApi;
 
-import org.jsoup.Connection;
-import org.jsoup.HttpStatusException;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
 
 import java.io.IOException;
+import java.util.LinkedList;
 
 
 @RestController
 @RequestMapping("/api/v1/bible-verses")
 public class BibleVerseController {
-    @GetMapping("/get-verse")
-    public Verse getVerse() throws IOException {
-        Document doc = Jsoup.connect("https://wol.jw.org/en/wol/b/r1/lp-e/nwtsty/1/1#study=discover&v=1:1:1")
-                .get();
-        doc.select("p").forEach(System.out::println);
+    @GetMapping("/get-verse/{book}/{chapter}")
+    public VerseList getVerse(
+            @PathVariable("book") String book,
+            @PathVariable("chapter") String chapter) throws IOException {
+        Document doc = Jsoup.connect(
+                AppSettings.mainVerseURL(book, chapter)
+        ).get();
+        LinkedList<String> verses = new LinkedList<String>();
```

Comment:
did some research and it looks like standard Java arrays are more performant than "LinkedLists". Is there a specific reason why you did this?

Reply:
You are correct. The reason for doing it this way was just a quick way to make it mutable, because raw arrays are immutable in the sense that you can't grow/shrink them

Reply:
The difference is extremely negligible. LinkedLists are also a WAY more performant implementation than python/js implementations

Reply:
Hmmmm.... Maybe but im pretty sure the array itself can be increased or decreased in size but the actual values in the array are immutable 

https://stackoverflow.com/questions/16125616/are-string-arrays-mutable

Reply:
> The difference is extremely negligible. LinkedLists are also a WAY more performant implementation than python/js implementations

all true just wanted to find out reasoning

Reply:
@j-koziel You want to use ArrayList instead?